### ISSUE TITLE
Improve the existing UI tests

### ISSUE THREAD

**Issue by PawelLipski** [VirtusLab/git-machete-intellij-plugin#274]
1. ~Use public artifacts of `com.virtuslab.ideprobe:junit-driver` and its dependencies instead of building&publishing them locally within the CI pipeline; blocked on VL's client approval wrt. publishing~ => done, migrated to publicly available Remote Robot on #317 

1. ~Possibly reuse the cached IntelliJ zips between binary compatibility check (currently in $HOME/.cache/intellij-verifier/) & UI tests (in /tmp/ideprobe/cache/<file-hash>, details TBD) so that they're not downloaded twice~ => done, as of #317, everything is stored in Gradle caches, no custom caches anymore

1. ~Exclude `com.virtuslab.gitmachete.frontend.actions.ideprobe.OpenTabAction` from production builds of the plugin if easily doable (or maybe not?)~ => ~**done on #311**~; also see #312;
**update: #311 overridden by #317**, custom actions for UI test finally no longer needed

1. ~As per @mkondratek comment: omit project selection window + tip of the day popup if possible~ => both resolved on #317 

1. ~Actually, IntelliJ has provided its own public alternative (?) for the proprietary ide-probe we're currently using: https://github.com/JetBrains/gradle-intellij-plugin/tree/master/examples/ui-test-example ; I don't think think that migration will make sense in the near future, but it's good to remember that such a solution exists now.~ => done on #317

**Comment by mkondratek** [comment]
From my comment https://github.com/VirtusLab/git-machete-intellij-plugin/pull/265#issuecomment-628900389:
> - omitting projects windows (the one where you chose the project :xd:) could speed up the tests
> - the same about tip popup within IDE

**Comment by mkondratek** [comment]
> \

_from plugin.xml_

Adding `(Open) Machete Tab` as an action to the toolbar is an option. 
Probably under `VCS` > `Git`.

**Comment by PawelLipski** [comment]
`s/to the toolbar/to the main menu`?

**Comment by mkondratek** [comment]
> `s/to the toolbar/to the main menu`?

yup, sorry
